src/main/java/web/reservation/controller/ReservationController.java

```diff
@@ -0,0 +1,68 @@
+package web.reservation.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import web.reservation.dto.ReservationRequestDto;
+import web.reservation.dto.ReservationResponseDto;
+import web.reservation.exception.ReservationException;
+import web.reservation.service.ReservationService;
+
+import javax.validation.Valid;
+import java.net.URI;
+import java.time.LocalTime;
+
+import static web.reservation.exception.ErrorCode.NOT_UNIT_OF_30_MINUTES;
+import static web.reservation.exception.ErrorCode.OUT_OF_BUSINESS_HOURS;
```

Comment:
Nit: 정적 import보다는 ErrorCode를 도메인별로 네이밍을 달리 해서 각 도메인별 ErrorCode 이넘으로만 import 시키고, 코드단에서는 xxxErrorCode.NOT_UNIT_OF_30_MINUTES로 접근하는건 어떠실까요??

각 상수값 하나하나 추가하면 import문이 방대해지기도 하고 또 사용하는 곳에서 ErrorCode의 상수를 이용한다고 명시하는게 좀 더 직관적일 것 같아서요 
```suggestion
import static web.reservation.exception.ReservationErrorCode;

throw new ReservationException(ReservationErrorCode.OUT_OF_BUSINESS_HOURS);
```

+) 전체 코멘트 읽어보시고 도메인별로 분리되어있는걸 합치는 방향에 대해서도 고민해주시면 좋을 것 같습니다~ 다만 이때도 각 상수값을 모두 import시키기보단 ErrorCode로 접근하는걸 권장드리긴 합니다